apisix/discovery/nacos.lua

```diff
@@ -211,7 +272,7 @@ local function get_base_uri()
         url = url .. '/'
     end
 
-    return url, username, password
+    return url, username, password, access_key, secret_key, authorization_type
```

Comment:
What about returning:
```
url, authorization_type, username, password, access_key, secret_key
or
url, authorization_type, access_key, secret_key
```

Reply:
这里的建议是 authorization_type 返回顺序往前提？

1、username, password
2、access_key, secret_key

1 和 2 是两两组合使用，用于支持 2 种不同的认证方式

Reply:
There is a typo in my suggestion...
It should be
```
url, authorization_type, username, password
or
url, authorization_type, access_key, secret_key
```

It seems that each authorization_type only uses two of them, so we can return only four values instead of six.

Reply:
好的，我改下